i18n/en.json

```diff
@@ -418,6 +418,10 @@
   "admin.file_upload.uploadFile": "Upload",
   "admin.files.images": "Images",
   "admin.files.storage": "Storage",
+  "admin.s3.s3Success": "Connection was successful: No errors were reported while testing the S3 access.",
```

Comment:
Propose we keep the error message as `Connection was successful`

Reply:
done